addons/web/static/src/js/views/control_panel/search/groupby_menu.js

```diff
@@ -157,6 +162,22 @@ var GroupByMenu = DropdownMenu.extend({
         }
     },
 
+    _computeMaxHeight: function() {
+        /*
+        * Compute the menu's max height to avoid
+        * it to be out of the screen.
+        *
+        * Taking into consideration:
+        *  - 46px for the Odoo banner
+        *  - 86px for the Control panel
+        *  - 24px for the Dashboard top padding
+        *  - 30px for the button's height
+        *  - 16px to avoid being too close from the limit
```

Comment:
all these seem so specific...

Reply:
I agree, I found a new solution based on the button's position.